scripts/prepare_production_deployment.sh

```diff
@@ -7,10 +7,11 @@ set -ev
 # - Security env variables are available.
 if [ -n "$VERSION_TAG" ] && [ -n "$PROD_DEPLOYMENT_HOOK_TOKEN" ] && [ -n "$PROD_DEPLOYMENT_HOOK_URL" ]
 then
+  APP_NAME="$(basename $(pwd))"
   curl --silent --output /dev/null --write-out "%{http_code}" -X POST \
      -F token="$PROD_DEPLOYMENT_HOOK_TOKEN" \
-     -F ref=master \
-     -F "variables[TRIGGER_RELEASE_COMMIT_TAG]=$VERSION_TAG" \
+     -F ref=main \
```

Comment:
This branch name is on dev-ops repo. Did they change their branch name from master to main?